### PR TITLE
Add create user view

### PR DESCRIPTION
**Pull request by willwull**
This PR adds a page in `/create-user` with a form to create a new user. It looks like this:

image

### CODE REVIEW

**Comment by willwull** [comment]
Just realized, is there supposed to be a "Company name" field?

**Comment by mictab** [comment]
@willwull Good catch. Yes, it should be there if "Company member" is selected.

**Comment by willwull** [comment]
Alright, will fix

File: app/containers/CreateUser/formUtils.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ * Utility function for converting form data to an object.
+ *
+ * @param {HTMLFormControlsCollection} formElements
+ */
+function formToObject(formElements) {
+  // Use reduce.call since formElements is not actually an array
+  const formData = [].reduce.call(formElements, (data, elem) => {
+    if (isValidElement(elem) && isValidValue(elem)) {
+      let value
+      if (elem.type == 'checkbox') {
```
Comment (mictab) [inline app/containers/CreateUser/formUtils.js]: Should be `===`?

**Comment by mictab** [comment]
Depends on:
https://github.com/studieresan/overlord/pull/41
https://github.com/studieresan/overlord/pull/39

File: app/containers/CreateUser/formUtils.js (hunk 1 above)
Comment (willwull) [reply app/containers/CreateUser/formUtils.js]: Ahh, you're correct, surprised we didn't have an ESLint rule for this!

Comment (mictab) [reply app/containers/CreateUser/formUtils.js]: If we don't we should probably add it!

**Comment by mictab** [comment]
Nice!

**Comment by willwull** [comment]
Alright, I fixed the errors that the `eqeqeq` rule introduced, so the project will compile now @mictab

**Pull request merged**